File: src/led/json/LEDJsonProcedure.java
package led.json;

import com.google.gson.annotations.SerializedName;

public class LEDJsonProcedure {
	@SerializedName("meta")
	public MetaInfo mMetaInfo;
	
	@SerializedName("led_state")
	public LEDStateArray[] mLEDStates;
}

File: src/main/java/led/json/LEDJsonProcedure.kt
package led.json

import com.google.gson.annotations.SerializedName

class LEDJsonProcedure {
    @SerializedName("meta")
    var mMetaInfo: MetaInfo? = null

    @SerializedName("led_state")
    lateinit var mLEDStates: Array<LEDStateArray?>
}